_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -14,6 +15,14 @@
 
 
 // ----------------------------
+// #PATHS
+// ----------------------------
+$static-path: '..' !default;
+$font-path: '#{$static-path}/fonts' !default;
+$image-path: '#{$static-path}/images' !default;
+
```

Comment:
Good enough for me, I'll keep it simple and see how folks use it in their application of the UXPL. Thanks!